### PR TITLE
sao: reduce menu width for big screens

### PR DESCRIPTION
**Pull request by manalaboutayeb**

### CODE REVIEW

**Comment by romainseon** [comment]
print screen?

**Comment by manalaboutayeb** [comment]
menu_big

**Comment by manalaboutayeb** [comment]
menu_small

File: theme/coog/offcanva.less [hunk 1]
```diff
@@ -12,15 +12,27 @@
 
   .row-offcanvas-left
   .sidebar-offcanvas {
-    left: calc(-25%);
+
+    @media (max-width: 1400px){
+      left: calc(-25%);
+    }
+    left: calc(-15%);
```
[the remark this answers is not in the document]
Comment (PierreCookie) [reply theme/coog/offcanva.less]: still here

**Pull request merged**